feature-styles/HY_P_Watercourse_ManMade.se

```diff
@@ -44,7 +44,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
      <se:Description>
-      <se:Title>waterbodies natural</se:Title>
+      <se:Title>Waterbodies natural</se:Title>
```

Comment:
For other cases with different geometries, the geometry was mentioned in the title - maybe align?